contracts/SureYouDo.sol

```diff
@@ -281,7 +281,7 @@ contract SureYouDo is AllowedTokensManager, ReentrancyGuard {
         if (
             platformCommissionPercentage > SydStructures.COMMISSION_PRECISION ||
             (isProAccount && platformCommissionPercentage < minPlatformCommissionProAccount) ||
-            (isProAccount == false && platformCommissionPercentage < minPlatformCommission)
```

Comment:
I don't think we can remove this check. If we do, it will revert when `isProAccount` is `true` and `platformCommissionPercentage < minPlatformCommission`, which we don't want.

Reply:
Yea, you are right 👍